package.json

```diff
@@ -40,15 +40,15 @@
     }
   },
   "devDependencies": {
-    "@typescript-eslint/eslint-plugin": "5.59.5",
-    "@typescript-eslint/parser": "5.59.5",
-    "eslint": "^8.40.0",
+    "@typescript-eslint/eslint-plugin": "5.60.0",
+    "@typescript-eslint/parser": "5.60.0",
+    "eslint": "^8.43.0",
     "eslint-find-rules": "4.1.0",
     "eslint-plugin-ava": "^14.0.0",
     "eslint-plugin-hildjj": "link:rules",
-    "eslint-plugin-jsdoc": "^44.2.3",
+    "eslint-plugin-jsdoc": "^46.2.6",
     "eslint-plugin-node": "^11.1.0",
-    "typescript": "5.0.4"
+    "typescript": "5.1.3"
```

Comment:
This is going to cause @typescript-eslint/parser to complain.  See https://github.com/typescript-eslint/typescript-eslint/pull/7088